tests/test_client.py

```diff
@@ -798,3 +807,10 @@ def test_remove_metadata(self, client, response_cases, mock_requests):
         # Check that the correct json with metadata id is sent
         json_expect = ["19b7230b-f88a-4217-b1c9-08daff938054"]
         assert mock_requests.call_args.kwargs["json"] == json_expect
+
+    def test_client_get_is_logged(self, client):
```

Comment:
We could add some description in this test on why we expect the logger to be called, something like 

```get_fails_with_valueerror_exception_is_logged```